default.json

```diff
@@ -18,7 +19,7 @@
   "lockFileMaintenance": {
     "enabled": true,
     "automerge": true,
-    "extends": ["schedule:weekly"]
+    "extends": ["schedule:weekly", "schedule:weekends"]
```

Comment:
Do we really want stuff scheduled to be created and possibly automerged on a weekend?